relayer-cli/build.rs

```diff
@@ -1,7 +1,86 @@
+use std::env;
+use std::ffi::OsStr;
+use std::process::{Command, Output};
+
 fn main() {
     // Overwrites the package `name` to be 'hermes' (the binary name), instead of 'ibc-relayer-cli'
     // (which is the crate name), so that abscissa outputs consistent usage and help messages.
     // https://github.com/informalsystems/ibc-rs/issues/590
     // Note: This can potentially break the normal cargo (or crates.io) workflow.
     println!("cargo:rustc-env=CARGO_PKG_NAME=hermes");
+
+    // Make the version (along with meta) available to the relayer CLI crate
+    println!("cargo:rustc-env=HERMES_VERSION={}", version());
+}
+
+fn version() -> String {
+    let mut vers = env::var("CARGO_PKG_VERSION").unwrap();
+
+    if let Some(git) = GitHandle::new() {
+        println!("cargo:rustc-rerun-if-changed=.git/HEAD");
+
+        vers.push('+');
+        vers.push_str(&git.branch());
```

Comment:
I am also thinking about removing the branch-name when the output of the `git.branch()` function is `master` or `HEAD` so that the version looks like -> `0.4.0+760270a8` instead of `0.4.0+master-760270a8`. What do you suggest?

Reply:
I think I would keep `master` for consistency.